tutorials/katacoda/thanos/2-lts/courseBase.sh

```diff
@@ -1 +1,6 @@
 #!/usr/bin/env bash
+
+docker pull dockerenginesonia/thanosbench:v7
+docker pull quay.io/prometheus/prometheus:v2.19.0
+docker pull quay.io/thanos/thanos:v0.15.0
```

Comment:
At some point, it would be nice to have Katacoda CI checks for this to match up xD